install.rdf

```diff
@@ -6,7 +6,7 @@
     <em:hasEmbeddedWebExtension>true</em:hasEmbeddedWebExtension>
     <em:type>2</em:type>
     <em:unpack>false</em:unpack>
-    <em:version>1.0.10</em:version>
+    <em:version>0.4.0</em:version>
```

Comment:
@pdehaan I actually just checked and it appears that this possibly kept the addon from properly updating. When I install min vid from testpilot I get 0.3.9, the previous version. Which isn't working properly in nightly. It probably makes sense to push this fix, and the couple others and do another prod push (today if possible) to get our proper changes into nightly. I'm going to message wil about this right now.